docs/contributing/contributing-releases/README.md

```diff
@@ -34,23 +34,28 @@ If this workflow run fails, or if we encounter an issue with an RC release, plea
 
 If sample validation passes, we can start the process of creating the final release.
 
-1. In the `radius-project/bicep` repo, in the release branch, change the `version.json` version to the new release number. Create a pull request and merge this change.
+1. In the `radius-project/bicep` repo, in the release branch, change the `version.json` version to the new release number. Create a pull request and merge this change into the bicep-extensibility branch.
 
 1. Go through steps 1-4 of "Creating an RC release" above, substituting the final release version instead of the RC version. For example, if the RC version number is `0.1.0-rc1`, the final release version would be `0.1.0`.
 
-1. After creating the pull request, there should be an automatically generated release notes comment. Create a new release note document in the [release-notes](../../release-notes/) directory. Follow the directory's README.md for instructions on how to create a new release note document. Include this file in the release version pull request. [Example](https://github.com/project-radius/radius/pull/6092/files)
+1. In this PR, create a new release note document in the [release-notes](../../release-notes/) directory. Follow the directory's README.md for instructions on how to create a new release note document. Include this file in the release version pull request. [Example](https://github.com/project-radius/radius/pull/6092/files)
 
-1. Cherry-pick the `version.yaml` changes and the release notes into the release branch. This will ensure that the version changes and release notes are included in the release branch. [Example](https://github.com/radius-project/radius/pull/6114/files)
+1. Create a PR to merge into the release branch (format: release/x.y) in the radius repo. Cherry-pick the `version.yaml` changes and the release notes from the previous step in this PR. This will ensure that the version changes and release notes are included in the release branch. [Example](https://github.com/radius-project/radius/pull/6114/files)
 
    ```bash
    git cherry-pick -x <COMMIT HASH>
    ```
+1. Verify release artifacts have been deployed to [GHCR](https://github.com/orgs/radius-project/packages)
 
 1. In the project-radius/docs repository, run the [Release docs](https://github.com/project-radius/docs/actions/workflows/release.yaml) workflow.
 
 1. In the project-radius/docs repository, run the [Release samples](https://github.com/project-radius/samples/actions/workflows/release.yaml) workflow.
 
-1. In the project-radius/radius repo, run the [Release verification](https://github.com/radius-project/radius/actions/workflows/release-verification.yaml) workflow.
+1. In the project-radius/radius repo, run the [Release verification](https://github.com/radius-project/radius/actions/workflows/release-verification.yaml) workflow. Run the workflow from the release branch (format: release/x.y) and use the Radius release version number being released.
```

Comment:
A bit confused on why we are running the main branch version of the workflow for RC but release branch version for full release.. can you help me understand this?

Reply:
> only if there is any difference between the two, will there be a difference

This is right. So if that's the motivation that we want to use point in time state of release verification workflow rather than latest, then that's what we should use for RC verification, else we signing up for surprises during full release verification - for example, if a bug for workflow was fixed in the main branch but wasn't captured in the release branch.

> When we verify RC branch, we run it against main. (there is no RC branch created.)

By this point, the release branch is created, so we can just use the workflow from that branch.

Reply:
Yes, updating it for RC release as well.